Iterable-iOS-SDK/IterableAPI.m

```diff
@@ -98,12 +98,21 @@ - (NSURL *)getUrlForAction:(NSString *)action
  */
 - (NSURL *)getUrlForGetAction:(NSString *)action withArgs:(NSDictionary *)args
 {
-    NSString *urlCombined = [NSString stringWithFormat:@"%@%@?api_key=%@", endpoint, action, self.apiKey];
-    //updated this to take in a dictionary are parse values
     
     
+    NSString *urlCombined = [NSString stringWithFormat:@"%@%@?api_key=%@", endpoint, action, self.apiKey];
+    
     for (NSString* paramKey in args) {
         NSString* paramValue = args[paramKey];
+
+        //Percent encode special characters
+        NSMutableCharacterSet* subSet = [[NSMutableCharacterSet alloc] init];
+        [subSet formUnionWithCharacterSet:[NSCharacterSet URLQueryAllowedCharacterSet]];
```

Comment:
also, you're creating this every time. make it once in the whole class. also, optionally split this stuff out into a `urlEncodeParam` function